### DISCUSSION TITLE
Не компилируется прошивка для 8266

### DISCUSSION THREAD

**Discussion by SkolExChita** [vvip-68/LedPanelWiFi#68] [Вопросы и ответы] [answered]
Всем привет.  при попытке компиляции выдает ошибку
 exit status 1
'LINEARBLEND_NOWRAP' was not declared in this scope
все версии всего проверил

**Comment by vvip-68** [comment, marked as answer]
Используйте FastLED версии 3.6 или 3.7

**Comment by SkolExChita** [comment]
Спасибо за ответ! 3,6 пробовал, всё равно не срабатывало...
![image](https://github.com/user-attachments/assets/bfc10167-6ff9-4f56-b2c3-73426ddaa666)
Белого заключения нет, значит опять что-то сделал не так?

**Reply by vvip-68** [reply]
Что вам здесь не нравится? Какое белое заключение вы ждете?
Иногда оранжевые буквы - это просто оранжевые буквы.

**Reply by SkolExChita** [reply]
понял Вас! Спасибо что возитесь со мной!
